build/emoji/make-emojis

```diff
@@ -140,6 +140,40 @@ File.open(File.join(project_root, 'utils', 'emoji.jsx'), 'w') do |f|
     puts "\nwrote #{emojis.length} emojis to utils/emoji.jsx"
 end
 
+webapp_server_dir = File.join(project_root, 'utils', 'server')
+Dir.mkdir(webapp_server_dir) unless Dir.exist?(webapp_server_dir)
+File.open(File.join(webapp_server_dir, 'emoji_data.go'), 'w') do |f|
+    f.write <<-eos
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+// This file is automatically generated via `make emojis`. Do not modify it manually.
+
+package model
+
+eos
+
+    emoji_dict = {}
+    emojis.each do |emoji|
+      emoji['aliases'].each do |emoji_alias|
+        # There are duplicity in some emojis, for example blonde_woman_light_skin_tone,
+        # so we get the last one of them because we only check for the alias/name
+        # in the server-side part
+        emoji_dict[emoji_alias] = %{"#{emoji_alias}": "#{emoji['filename']}"}
+      end
+    end
+    f.write("var SystemEmojis = map[string]string{#{emoji_dict.values.join(', ')}}\n")
+
+    puts "\nwrote #{emojis.length} emojis to utils/server/emoji_data.go"
+end
+
+# If SERVER_DIR is defined we can update the file emoji_data.go in
+# the server directory
+if ENV.key?('SERVER_DIR')
+  server_dir = File.join(ENV['SERVER_DIR'], 'model')
+  FileUtils.cp(File.join(webapp_server_dir, 'emoji_data.go'), server_dir)
```

Comment:
Would it make sense to move this file instead of copying it? I'd like to avoid leaving around an extra file that might get checked in by accident.

Reply:
Sure. It makes sense. I didn't know if the file was need for something :)